tests/test_s3_parquet.py

```diff
@@ -70,6 +70,27 @@ def test_read_parquet_filter_partitions(path, use_threads):
     assert df2.c2.astype(int).sum() == 0
 
 
+def test_read_parquet_table(path, glue_database, glue_table):
+    df = pd.DataFrame({"c0": [0, 1, 2], "c1": [0, 1, 2], "c2": [0, 0, 1]})
+    wr.s3.to_parquet(df, path, dataset=True, database=glue_database, table=glue_table)
+    df_out = wr.s3.read_parquet_table(table=glue_table, database=glue_database)
+    assert df_out.shape == (3, 3)
```

Comment:
What is the point of this test? Was it just that we did not have one for plain `read_parquet_table`?

Reply:
Yes, that's just a test case for basic scenario.